src/integ-test/bdd-test/features/steps/steps.py

```diff
@@ -70,7 +70,9 @@ def step_impl(context, elem, value, action):
     elif action == "paVerifyPaymentNoticeRes":
         pa_verify_payment_notice_res = getattr(context, "paVerifyPaymentNoticeRes")
         pa_verify_payment_notice_res = utils.manipulate_soap_action(pa_verify_payment_notice_res, elem, value)
-        response_status_code = utils.save_soap_action(utils.get_rest_mock_ec(context), "paVerifyPaymentNotice",
+        setattr(context, action, pa_verify_payment_notice_res)
```

Comment:
Why we need to save `paVerifyPaymentNoticeRes` response ?
And i think we should change name of `save_soap_action` in to `send_mock_requests` or something like that.

Reply:
`paVerifyPaymentNoticeRes` is manipulated again in the next step.
ok for method rename, I suggest `prepare_mock_requests`